Sources/ITwinMobile/ITMAuthorizationClient.swift

```diff
@@ -188,11 +210,6 @@ open class ITMAuthorizationClient: NSObject, AuthorizationClient, OIDAuthStateCh
     
     /// Calls the onUserStateChanged callback, if that callback is set.
     open func raiseOnUserStateChanged() {
```

Comment:
This whole function should go away, but perhaps I should clean things up further after the initial release is done, since unused stuff won't prevent it from working.

Reply:
Done